themes/default/content/blog/organizing-aws-accounts-with-pulumi/index.md

```diff
@@ -0,0 +1,289 @@
+---
+title: "Organizing Aws Accounts With Pulumi"
+date: 2023-01-16
+meta_desc: Learn how you can use Pulumi to manage your AWS Organizations hierarchy and automatically create member accounts.
+meta_image: meta.png
+authors:
+    - praneet-loke
+tags:
+    - aws-organizations
+---
+
+In an Enterprise organization, an IT Self Service Vending Machine allows employees to quickly and easily request and receive access to pre-approved cloud resources. Behind the scenes, Pulumi programs may orchestrate any of the requisite resources. We will look at an example of using Pulumi to create an AWS child account, within an AWS Organization.
+
+<!--more-->
+
+One might envision a developer requesting a new AWS account to be added entirely through a Pull Request in a git repository either for their own use or on behalf of a team. This scenario is not just for enterprises, however. Teams of any size should organize their AWS accounts using the least-privileged access principle.
```

Comment:
I have to admit I am not very fond of this opener a whole lot.

Reply:
I think this is fine!